src/components/Content/ContentHeader/ExperimentButtons/Container.js

```diff
@@ -0,0 +1,87 @@
+// CORE LIBS
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { withRouter, useParams } from 'react-router-dom';
+
+// COMPONENTS
+import ExperimentButtons from './index';
+
+// ACTIONS
+import { fetchExperimentDeployStatusRequest } from '../../../../store/experiment/actions';
```

Comment:
@shiguemi32 aprova pra adicionar um catch em algum no service de consulta de status de deployment, por favor:
![catch](https://user-images.githubusercontent.com/1791381/92475462-ced92a00-f1b3-11ea-99dc-cc8cf118c9aa.PNG)

este warning polui o console.

Reply:
Não é possível remover esse erro do console.

https://stackoverflow.com/questions/16372271/jquery-ajax-how-to-prevent-404-errors-spam-in-chrome-devtools